src/plays/react-todo-app/components/TodoList.jsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react'
+import Todo from './Todo';
+
+const TodoList = ({ todos, setTodos, filtered }) => {
+    return (
+        <div className="todo-container">
+            <ul className="todo-list">
+                {filtered.map((todo) => (
```

Comment:
We can show a message if the filtered is empty... like no TODO found matching the criteria.